samples/bluetooth/throughput/README.rst

```diff
@@ -55,15 +55,21 @@ By default, the following connection parameter values are used:
 Changing connection parameter values
 ====================================
 
-You can experiment with different connection parameter values by reconfiguring the values using menuconfig and then compiling and programming the sample again to at least one of the boards.
+You can experiment with different connection parameter values by reconfiguring the values using the :ref:`zephyr:shell_api` interface before running a test.
+The following parameters can be adjusted:
+   * PHY
+   * LE Data Length
+   * LE Connection interval
 
 .. note::
    In a *Bluetooth* Low Energy connection, the different devices negotiate the connection parameters that are used.
    If the configuration parameters for the devices differ, they agree on the lowest common denominator.
 
    By default, the sample uses the fastest connection parameters.
-   If you change them to lower values, it is sufficient to program them to one of the boards.
-   If you were to change them to higher values, you would need to program both boards again.
+   You can change them to different valid values without a need to program the boards again.
+
+.. note::
+   When LE Connection Interval is set to high values and the PHY or the Data Length needs to be changed in the next test then the PHY Update procedure or Data Length Update procedure can take several seconds.
```

Comment:
When you have set the LE Connection Interval to high values and need to change the PHY or the Data Length in the next test, the PHY...

Reply:
Done